front/src/index.tsx

```diff
@@ -31,9 +33,13 @@ root.render(
             <AuthAutoRouter />
             <AppThemeProvider>
               <SnackBarProvider>
-                <StrictMode>
-                  <App />
-                </StrictMode>
+                <DialogProvider>
+                  <SpreadsheetImportProvider>
```

Comment:
I think this should not be added to all pages, only the one that need the import feature